marklogic-data-hub/src/test/java/com/marklogic/hub/flow/TemporalWriteTest.java

```diff
@@ -17,18 +19,21 @@
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 
+import java.io.ByteArrayInputStream;
+import java.io.InputStream;
+import java.util.stream.Stream;
+
 import static com.marklogic.client.io.DocumentMetadataHandle.Capability.*;
 import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
 
 public class TemporalWriteTest extends AbstractHubCoreTest {
 
+    private InputEndpoint.BulkInputCaller bulkInputCaller;
+
     @BeforeEach
     void setUp() {
-        // In theory, a data-hub-developer should be able to deploy these temporal resources. But when run in the
-        // full suite, this test sometimes fails, presumably due to changes made by a previous test to the databases.
-        // Since the purpose of this test is to verify that temporal documents can be written as an operator and not
-        // to verify that a developer can deploy these things (we have other tests for that), admin is used here
-        runAsAdmin();
+        runAsDataHubDeveloper();
```

Comment:
Let's not change what was here - as the comments noted, data-hub-developer should work here. But the test was failing intermittently. Since the purpose of this test isn't to verify that a data-hub-developer can deploy temporal resources, it's fine to run it as admin.

Reply:
Done